core/unit_test/incremental/Test05_ParallelReduce_RangePolicy.hpp

```diff
@@ -54,7 +54,6 @@ namespace Test {
 
 using value_type       = double;
 constexpr double value = 0.5;
-const int num_elements = 10;
 
 struct ReduceFunctor {
```

Comment:
Given that trivially copyable functors go through a different code path than non-trivially copyable ones, we should add a test for a non trivially copyable one, such as the one at https://github.com/nliber/kokkos/commit/d72418b5f2617fc394d1375d646c0b345ccd03ea.

We should add these in general, not just for reductions.

Reply:
Done.